src/blocks/HeaderBlockBrandPivot/index.tsx

```diff
@@ -215,69 +240,87 @@ export const Header: React.FC<{ story: GlobalStory } & HeaderBlockProps> = (
                     preventInverse={props.inverse_colors && isOpen}
                   />
 
+                  <MediaQuery query="(min-width: 1001px)">
+                    <ContextContainer>
+                      {(context) => (
+                        <LogoLink
+                          href={
+                            '/' + (context.lang === 'sv' ? '' : context.lang)
+                          }
+                        >
+                          <HedvigLogotype width={98} />
+                        </LogoLink>
+                      )}
+                    </ContextContainer>
+                  </MediaQuery>
+                </RightContainer>
```

Comment:
I was confused by this since it's to the left now haha, maybe change the name?
```suggestion
                </LeftContainer>
```